docs/tutorials/implement_new_games/index.rst

```diff
@@ -53,12 +48,17 @@ has :code:`flip`, which flips a :code:`C` to a :code:`D` and vice versa!)
 A simple rock-paper-scissors action class would look like so::
 
     >>> from enum import Enum
-    >>> class RPSAction(Enum):
+    >>> from functools import total_ordering
+    >>> @total_ordering
+    ... class RPSAction(Enum):
     ...     """Actions for Rock-Paper-Scissors."""
     ...     R = 0  # rock
     ...     P = 1  # paper
     ...     S = 2  # scissors
     ...     
+    ...     def __lt__(self, other):
```

Comment:
Why do we need this? Are we ever relying on R < P < S ? Also it's technically a circular ordering, since S < R.

Reply:
I mentioned a more full reason here: https://github.com/Axelrod-Python/Axelrod/pull/1418#issuecomment-1537077559 and also in the paragraph below this code block in the docs page itself.

but outside of interoperability between games (which we may or may not want) it's there for best practice (and to avoid user confusion if someone creates a game with a strategy containing a line like `if action == P`), and to keep consistency with the library itself [it has been implemented in exactly the same way as the IPD `Action` class implements ordering of actions](https://github.com/Axelrod-Python/Axelrod/blob/784cdd361bc40119b828e97801d0bb58db3a5669/axelrod/action.py#L21-L33). I can change it to just an `__eq__` method if you prefer and remove the `@total_ordering`.